### PR TITLE
feat: next new standard and actions pages 4.1-4.2-4.4

### PR DESCRIPTION
**Pull request by danielaiozzo**

### CODE REVIEW

**Comment by danielaiozzo** [comment]
@Fupete ready for merge; please exclude [this commit](https://github.com/italia/designers.italia.it/pull/663/commits/65f639a8aa22b0721c7ff0f85f8bff5fc984d3b5), I've just opened [a new PR](https://github.com/italia/designers.italia.it/pull/669) for it

File: src/data/content/norme-e-riferimenti/linee-guida-di-design/requisito-4-1/azione-1.yaml [hunk 1]
```diff
@@ -0,0 +1,357 @@
...
+              icon: sprites.svg#it-more-items
+              size: sm
+              color: secondary
+          list:
+            isShare: true
+
+      # CARD CHILDREN ACTION
+      - title: "Progettare con qualità: l’accessibilità al centro del design system del Paese"
+        headingLevel: 4
+        img: /images/community/articoli-di-approfondimento/2021-medium-zaia-processo.webp
```
Comment (Fupete) [inline src/data/content/norme-e-riferimenti/linee-guida-di-design/requisito-4-1/azione-1.yaml:298]: @danielaiozzo wrong image url, please update

File: src/data/content/norme-e-riferimenti/linee-guida-di-design/requisito-4-1/azione-1.yaml [hunk 2]
```diff
@@ -0,0 +1,357 @@
...
+        externalLink:
+          label: Leggi su medium
+          screenReaderText: " l'articolo completo"
+          icon:
+            icon: sprites.svg#it-external-link
+            size: xs
+        moreInfo: di Daniele Tabellini e Fabrizio Caccavello
+        chips:
+          - color: secondary
+            label: Processo
```
Comment (Fupete) [inline src/data/content/norme-e-riferimenti/linee-guida-di-design/requisito-4-1/azione-1.yaml:315]: @danielaiozzo this card has different tags on the current website (/community), please update it

Comment (danielaiozzo) [reply src/data/content/norme-e-riferimenti/linee-guida-di-design/requisito-4-1/azione-1.yaml]: done, please check

**Review by Fupete — APPROVED** [review]

**Pull request merged**